spec/jobs/data_warehouse/table_summary_stats_export_job_spec.rb

```diff
@@ -30,7 +31,8 @@
     allow(Identity::Hostdata).to receive(:aws_region).and_return('us-west-1')
     allow(IdentityConfig.store).to receive(:s3_data_warehouse_bucket_prefix).
       and_return(s3_data_warehouse_bucket_prefix)
-
+    allow(IdentityConfig.store).to receive(:data_warehouse_enabled).
+      and_return(true)
```

Comment:
After looking at this for a bit it seems confusing to me, I have let blocks in the context to assign true or false based on the test, why would I pass a variable and check against itself rather than an explicit boolean value?